src/System.Diagnostics.Process/src/System/Diagnostics/Process.Windows.cs

```diff
@@ -160,8 +152,6 @@ private bool WaitForExitCore(int milliseconds)
                 {
                     _error.WaitUtilEOF();
                 }
-
-                ReleaseProcessHandle(handle);
```

Comment:
I couldn't find how they would interact, but as I'm not 100% sure I'll make it identical.